site/styles.css

```diff
@@ -507,8 +507,8 @@ figcaption {
   text-shadow: 2px 0 #fff, -2px 0 #fff, 0 2px #fff, 0 -2px #fff, 1px 1px #fff, -1px -1px #fff, 1px -1px #fff, -1px 1px #fff;
   background-color: white;
   background-image: url('/assets/img/ecosystem-checks.svg');
-  background-size: 40%;
-  background-position: right 0px bottom 0px; 
+  background-size: 30%;
```

Comment:
I don't like it at this size — I tweaked it pretty much all the way from 10% to 100% and the 40% was the sweet spot where the faint green checkmark was still visible.

Reply:
Also, now you've moved it around so much there is too much padding at the bottom of the `.feature`.

Reply:
I reverted the size to 40% but had to increase the offset a bit more to avoid overlapping with the text. This works for me if you're OK with it — the original asymmetrical or diagonal placement of the checkmarks looked off to me, hence the change. I actually quite like how things look now, not too subtle, not too distracting. 

![image](https://github.com/user-attachments/assets/7da3cb89-8052-44b6-bcd9-38af9b413571)

Reply:
It kind of just looks like you've cut the light-green checkmark off now? 😅  (The original version was WAY flashier than any of commits I send to the cloud, so just know I toned it down for you)

Reply:
Let's chat when we meet in 7 mins ... 😁 I'm sure we can figure it out. The light green checkmark being cut off is OK-ish, I think. Would be solved by sizing to 30%! 😛

Reply:
Reverted to 40%.